src/axom/sidre/docs/sphinx/mfem_sidre_datacollection.rst

```diff
@@ -90,3 +130,63 @@ example of how a simulation code might utilize the automatic reconstruction logi
   * There must be a coordinate set named ``coords``
   * There must be a topology named ``mesh`` with corresponding attributes stored in a field named ``mesh_material_attribute``
   * There must be a topology named ``boundary`` with corresponding attributes stored in a field named ``boundary_material_attribute``
+
+Mixed-material Fields
+---------------------
+
+The Conduit Blueprint provides support for mixed-material simulations through its "material set" construct.
+Material metadata (stored in ``mfem::GridFunction`` s) can be registered in the ``DataCollection`` like
```

Comment:
nitpick: the space between ``mfem::GridFunction``  and s is slightly awkward.  does it render properly if it is removed?